.github/workflows/linters.yml

```diff
@@ -0,0 +1,23 @@
+name: django CMS linters.yml
+
+on: [push]
+
+jobs:
```

Comment:
I'm sure that we had isort for the packages too.

Reply:
Agreed that isort will be tackled separately, right now it's more important to unblock the fact that no tests are running at all in Travis.